docs/configuration.md

```diff
@@ -12,7 +12,6 @@ Supported flags are as follows (result of running `./flagd start --help`):
   -p, --port int32                Port to listen on (default 8013)
   -c, --server-cert-path string   Server side tls certificate path
   -k, --server-key-path string    Server side tls key path
-  -s, --service-provider string   Set a service provider e.g. http or grpc (default "http")
   -a, --sync-provider-args        Sync provider arguments as key values separated by =
   -d, --socket-path string        Set the flagd socket path. With grpc the service will become available on this address. With http(s) the grpc-gateway proxy will use this address internally
```

Comment:
`With http(s) the grpc-gateway proxy will use this address internally`

Is that still true?

Reply:
removed